tools/double-spend/double-spend.go

```diff
@@ -12,53 +13,107 @@ import (
 		fmt.Println(err)
+		return
+	}
+
+	var myOutputID string
+	var confirmed bool
+	// wait for the funds
+	for i := 0; i < 10; i++ {
+		time.Sleep(5 * time.Second)
+		resp, err := clients[0].GetUnspentOutputs([]string{myAddr.String()})
+		if err != nil {
+			fmt.Println(err)
+			return
+		}
+		fmt.Println("Waiting for funds to be confirmed...")
+		for _, v := range resp.UnspentOutputs {
+			if len(v.OutputIDs) > 0 {
+				myOutputID = v.OutputIDs[0].ID
+				confirmed = v.OutputIDs[0].InclusionState.Confirmed
+				break
+			}
+		}
+		if myOutputID != "" && confirmed {
+			break
+		}
 	}
 
-	const genesisBalance = 1000000000
-	genesisWallet := wallet.New(genesisSeedBytes)
-	genesisAddr := genesisWallet.Seed().Address(0)
-	genesisOutputID := transaction.NewOutputID(genesisAddr, transaction.GenesisID)
+	if myOutputID == "" {
+		fmt.Println("Could not find OutputID")
+		return
+	}
+
+	if !confirmed {
+		fmt.Println("OutputID not confirmed")
+		return
+	}
+
+	out, err := transaction.OutputIDFromBase58(myOutputID)
+	if err != nil {
+		fmt.Println("malformed OutputID")
+		return
+	}
 
-	// issue transactions which spend the same genesis output in all partitions
+	// issue transactions which spend the same output
 	conflictingTxs := make([]*transaction.Transaction, 2)
 	conflictingTxIDs := make([]string, 2)
 	receiverSeeds := make([]*wallet.Seed, 2)
+
+	var wg sync.WaitGroup
 	for i := range conflictingTxs {
+		wg.Add(1)
+		go func(i int) {
+			defer wg.Done()
 
-		// create a new receiver wallet for the given conflict
-		receiverSeeds[i] = wallet.NewSeed()
-		destAddr := receiverSeeds[i].Address(0)
+			fmt.Println(i)
 
-		tx := transaction.New(
-			transaction.NewInputs(genesisOutputID),
-			transaction.NewOutputs(map[address.Address][]*balance.Balance{
-				destAddr: {
-					{Value: genesisBalance, Color: balance.ColorIOTA},
-				},
-			}))
-		tx = tx.Sign(signaturescheme.ED25519(*genesisWallet.Seed().KeyPair(0)))
-		conflictingTxs[i] = tx
+			// create a new receiver wallet for the given conflict
+			receiverSeeds[i] = wallet.NewSeed()
+			destAddr := receiverSeeds[i].Address(0)
 
-		valueObject := valuepayload.New(valuepayload.GenesisID, valuepayload.GenesisID, tx)
+			tx := transaction.New(
+				transaction.NewInputs(out),
+				transaction.NewOutputs(map[address.Address][]*balance.Balance{
+					destAddr: {
+						{Value: 1337, Color: balance.ColorIOTA},
+					},
+				}))
+			tx = tx.Sign(signaturescheme.ED25519(*myWallet.Seed().KeyPair(0)))
+			conflictingTxs[i] = tx
 
-		// issue the value object
-		txID, err := client.SendPayload(valueObject.Bytes())
-		if err != nil {
-			fmt.Println(err)
-		}
-		conflictingTxIDs[i] = txID
-		fmt.Printf("issued conflict transaction %s\n", txID)
-		//time.Sleep(7 * time.Second)
+			valueObject := valuepayload.New(valuepayload.GenesisID, valuepayload.GenesisID, tx)
+
+			// // issue the tx
+			conflictingTxIDs[i], err = clients[i].SendPayload(valueObject.Bytes())
```

Comment:
`conflictingTxIDs` should be `conflictingMsgIDs`. Otherwise it's a bit misleading

The comment has 2 `//`.